File: 백준/Bronze/1919. 애너그램 만들기/애너그램 만들기.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.util.Arrays;

public class Main {

    private static char[] first;
    private static char[] second;
    private static int[][] arr = new int[2][26];

    public static void main(String[] args) throws IOException {

        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

        first = br.readLine().toCharArray();

        for (char cur : first) {
            arr[0][cur - 'a']++;
        }

        second = br.readLine().toCharArray();
        for (char cur : second) {
            arr[1][cur - 'a']++;
        }

        int count = 0;
        for(int i = 0; i < 26; i++) {
            count += Math.abs(arr[0][i] - arr[1][i]);
        }

        System.out.println(count);
    }
}

File: 백준/Bronze/1919. 애너그램 만들기/애너그램 만들기.py
from collections import Counter

first = input().strip()
second = input().strip()

count_first = Counter(first)
count_second = Counter(second)

count = sum(abs(count_first[char] - count_second[char]) 
            for char in set(count_first.keys()).union(set(count_second.keys())))

print(count)
